build/detect.mk

```diff
@@ -2,13 +2,16 @@
 
 UNAME_S := $(shell uname -s)
 UNAME_M := $(shell uname -m)
+ETC_ISSUE := $(shell cat /etc/issue)
```

Comment:
This is fragile. Anybody can put any text in here, see `man 5 issue`.
I'd rather use `/etc/os-release` (which is standardized, see `man 5 os-release`)

Reply:
I did not aware. Will refer os-release.